### DISCUSSION TITLE
Context menu/item corners never square

### DISCUSSION THREAD

**Discussion by adamp01** [black7375/Firefox-UI-Fix#487] [General]
Hey @black7375 - On esr102 (Waterfox G5.0.2) and MacOS the context menu and context menu items never seem to have square corners. This is using v6.3.0.

Any ideas for why this might be happening? I checked the css for the menupopup and it is applied and active, but it seems the border-radius property has no effect on this element.

<img width="893" alt="image" src="https://user-images.githubusercontent.com/47503375/196920403-5f41cb3a-1a5f-48bd-a7e0-8d3ff24dbf7c.png">

**Comment by adamp01** [comment]
Looks like it's just related to native context menu: `widget.macos.native-context-menus`. Unfortunately setting that to false makes the menu look a bit nasty.

**Comment by black7375** [comment]
It seems to be a bug, so I'll deal with it in issue #488

In pure firefox, it now looks like this:

![image](https://user-images.githubusercontent.com/25581533/197090179-88337ef3-57d1-4c2e-a0ca-6432c6ffb449.png)

---

I'll set up Waterfox tonight and check it out.

**Comment by black7375** [comment]
@adamp01 The background color or separator shape of the menu pop-up looks similar to the windows 10.

How can I get Waterfox `v5.0.2`?
It seems to be able to use only `v5.0.1` yet.
- https://www.waterfox.net/download/
